spec/amp-framework-hosting.md

```diff
@@ -7,11 +7,33 @@ You can host the AMP framework and components from your own server or CDN. This
 - deliver the AMP framework in regions where `cdn.ampproject.org` may not be available.
 - serve AMP pages and the framework from the same host, potentially improving content delivery times.
 
-If you follow the recommendations made here, your AMP pages will still pass [validation](https://amp.dev/documentation/guides-and-tutorials/learn/validation-workflow/validate_amp/). When your AMP page is served from your own servers, the host you specify will be used to download the framework. When your AMP page is served from an [AMP cache](https://github.com/ampproject/amphtml/blob/master/spec/amp-cache-guidelines.md#references), the URLs to the AMP framework will be re-written to download from the AMP cache. This is important to keep in mind if you modify the AMP framework; your AMP pages could behave differently when displayed in a viewer (e.g. the [Google AMP Viewer](https://search.google.com/test/amp)) than when directly accessed from your host.
+The AMP Project is looking into options for [validation](https://amp.dev/documentation/guides-and-tutorials/learn/validation-workflow/validate_amp/) of AMP pages that use an AMP framework hosted outside of `cdn.ampproject.org`. As of April 2020, these AMP pages do not pass validation.
+
+## Versioning
+
+This document makes frequent use of the terms "version" and "runtime version (rtv)". These two versions are slightly different. When the AMP framework is built, it is assigned a 13-digit _version_. When the AMP framework is served, a config number prefixes the version, resulting in a 15-digit _runtime version_. The build system and runtime enforce these version formats.
+
+The AMP Project has adopted the following conventions:
+
+- Version: the commit time of the last commit in the active branch
+
+  ```
+  TZ=UTC git log -1 --pretty="%cd" --date=format-local:%y%m%d%H%M%S
+  ```
+
+  with a trailing single-digit build code (almost always `0`). This version corresponds to the release versions found on [github.com/ampproject/amphtml/releases](https://github.com/ampproject/amphtml/releases).
+
+- Runtime version (rtv): the version prefixed by a 2-digit config code:
+
+  - Canary: `00`
+  - Production: `01`
+  - Experiment: `02`, `03`, `04`, etc.
```

Comment:
00 - Experimental
01 - Stable
02 - Control
03 - Beta
04 - Nightly
05 - Nightly-Control

/cc @danielrozenberg for opinions on RTV explanation and how much to include here versus https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md

Reply:
I would be very happy to see this "version vs. rtv" explanation in the release documentation. It's only here because i couldn't find any public-facing documentation (outside of code comments and github issue/pr comments) about what an rtv is.

Reply:
Again I'll defer to Daniel here, I believe he's working on some public facing documentation updates to go with Nightly releases as well.

Reply:
This information should definitely either be here, or split into its own file. For now I think it's fine to put it here and link to this section if this seems relevant in other files (such as release-schedule.md)

Somewhere in this document (possibly, but not necessarily in this section) you should mention that at the very least to self-serve AMP one must have the `01` files - anything else is only for those interested in performing various A/B tests

Also, adding to @rcebulko's list:
* `10`, `11`, `12` - Experiments A, B, and C as defined in https://github.com/ampproject/amphtml/blob/master/build-system/global-configs/experiments-config.json (there's a README.md file in the same directory) with their `environment` set to `AMP`
* `21`, `23`, `25` - - Experiments A, B, and C with their `environment` set to `INABOX` (ads); `20`, `22`, and `24` can be used as prefixes for control populations (i.e., same traffic volume, but binary equivalent to `01`)

Reply:
Done